Annex60/Fluid/Sensors/TemperatureTwoPort.mo

```diff
@@ -63,8 +64,9 @@ equation
   end if;
   // Output signal of sensor
   if dynamic then
-    if enableThermalLosses then
-      der(T) = (TMed-T)*k*tauInv + (TAmb-T)*tauLossInv;
+    if transferHeat then
+      // Note that Dymola will remove the term (TAmb-T)*tauHeaTraInv if tauHeaTraInv=0.
```

Comment:
My analysis was wrong. The term is removed not because tauHeaTraInv=0, but
because transferHeat=false causes the other branch of the if-then construct
to be evaluated.
I removed the comment in
https://github.com/iea-annex60/modelica-annex60/commit/a812901d3045601cc79dadbc2437676d2f11da92

On Sat, Jun 20, 2015 at 10:16 AM, Filip <notifications@github.com> wrote:

> In Annex60/Fluid/Sensors/TemperatureTwoPort.mo
> <https://github.com/Mathadon/modelica-annex60/pull/3#discussion_r32886163>
> :
>
> > @@ -63,8 +64,9 @@ equation
> >    end if;
> >    // Output signal of sensor
> >    if dynamic then
> > -    if enableThermalLosses then
> > -      der(T) = (TMed-T)*k*tauInv + (TAmb-T)*tauLossInv;
> > +    if transferHeat then
> > +      // Note that Dymola will remove the term (TAmb-T)*tauHeaTraInv if tauHeaTraInv=0.
>
> That's odd since Dymola provides the option to change the parameter value
> after compilation and if you then change tauLossInv to be not equal to
> zero, the equation would be wrong?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/Mathadon/modelica-annex60/pull/3/files#r32886163>.
>